pkg/reconciler/common/job_test.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"testing"
+
+	"knative.dev/operator/pkg/apis/operator/v1alpha1"
+
+	"go.uber.org/zap"
+	batchv1 "k8s.io/api/batch/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+
+	util "knative.dev/operator/pkg/reconciler/common/testing"
+)
+
+const (
+	StorageVersionMigration         = "storage-version-migration"
+	StorageVersionMigrationEventing = "storage-version-migration-eventing"
+	StorageVersionMigrationServing  = "storage-version-migration-serving"
```

Comment:
I don't think we need to care about these, so I would omit them. Who's to say where the component name might show up in the name? I don't think we need to worry about that.